project/HmrcBuild.scala

```diff
@@ -38,7 +38,7 @@ private object BuildDependencies {
   import play.PlayImport._
   import play.core.PlayVersion
 
-  val httpVerbsVersion = "2.2.1"
+  val httpVerbsVersion = "3.0.0"
```

Comment:
minor note - can't this be inlined? I don't understand why most of our services have adopted this approach when there is only one usage :P